### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by WellingtonAssuncao** [Mg30/odds-portal-scraper#11]
Hello, 
I installed the program as explained, but when I go to consult with the command that you indicate, the following error appears:

2023-08-14T18:13:40.342Z - INFO - start scraping: https://www.oddsportal.com/soccer/england/premier-league-2019-2020/results/
2023-08-14T18:13:49.112Z - INFO - start scraping: https://www.oddsportal.com/soccer/england/premier-league-2020-2021/results/
2023-08-14T18:13:55.819Z - INFO - start scraping: https://www.oddsportal.com/soccer/england/premier-league-2021-2022/results/
2023-08-14T18:14:31.547Z - ERROR - [{"status":"rejected","reason":{"name":"TimeoutError"}},{"status":"rejected","reason" :{"name":"TimeoutError"}},{"status":"rejected","reason":{"name":"TimeoutError"}}]
2023-08-14T18:14:31.548Z - INFO - closing browser

**Comment by WellingtonAssuncao** [comment]
> Hi,
> 
> which version are you running ? what was the command ?

Version 2.3.0

odds-portal historic premier-league 2019 2021 --local ./data --odds-format eu

**Comment by Mg30** [comment]
It works on my side maybe your IP is blocked if you have been using it a lot.

**Comment by Lukeyone** [comment]
I got error too 

2023-10-03T12:52:23.504Z - INFO - start scraping: https://www.oddsportal.com/soccer/england/premier-league-2019-2020/results/
2023-10-03T12:52:36.037Z - INFO - start scraping: https://www.oddsportal.com/soccer/england/premier-league-2020-2021/results/
2023-10-03T12:52:36.089Z - INFO - https://www.oddsportal.com/football/england/premier-league-2019-2020/results/#/page/1
2023-10-03T12:52:36.100Z - INFO - setting odds as 'EU Odds'
2023-10-03T12:52:38.138Z - WARN - Odds have not been changed: May be because odds are already set in the required format
2023-10-03T12:52:38.139Z - INFO - fetching links
2023-10-03T12:52:42.778Z - INFO - start scraping: https://www.oddsportal.com/soccer/england/premier-league-2021-2022/results/
2023-10-03T12:52:42.820Z - INFO - https://www.oddsportal.com/football/england/premier-league-2020-2021/results/#/page/1
2023-10-03T12:52:42.828Z - INFO - setting odds as 'EU Odds'
2023-10-03T12:52:44.870Z - WARN - Odds have not been changed: May be because odds are already set in the required format
2023-10-03T12:52:44.870Z - INFO - fetching links
2023-10-03T12:52:47.425Z - INFO - https://www.oddsportal.com/football/england/premier-league-2021-2022/results/#/page/1
2023-10-03T12:52:47.434Z - INFO - setting odds as 'EU Odds'
2023-10-03T12:52:49.475Z - WARN - Odds have not been changed: May be because odds are already set in the required format
2023-10-03T12:52:49.475Z - INFO - fetching links
Failed to write data to data\27 Jul 2020,-Arsenal-Watford.json: [Error: ENOENT: no such file or directory, open 'F:\Downloads\odds-portal-scraper-main\data\27 Jul 2020,-Arsenal-Watford.json'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'F:\\Downloads\\odds-portal-scraper-main\\data\\27 Jul 2020,-Arsenal-Watford.json'
}
2023-10-03T12:52:49.625Z - ERROR - extracting data: Error: ENOENT: no such file or directory, open 'F:\Downloads\odds-portal-scraper-main\data\27 Jul 2020,-Arsenal-Watford.json'
Failed to write data to data\24 May 2021,-Arsenal-Brighton.json: [Error: ENOENT: no such file or directory, open 'F:\Downloads\odds-portal-scraper-main\data\24 May 2021,-Arsenal-Brighton.json'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'F:\\Downloads\\odds-portal-scraper-main\\data\\24 May 2021,-Arsenal-Brighton.json'
}


basically like this the whole time when running odds-portal historic premier-league 2019 2021 --local ./data --odds-format eu

**Comment by Lukeyone** [comment]
I think you forgot the data folder
